integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoQueryRunner.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.integration.test.trino;
+
+import static io.trino.cli.ClientOptions.OutputFormat.CSV;
+
+import io.airlift.units.Duration;
+import io.trino.cli.Query;
+import io.trino.cli.QueryRunner;
+import io.trino.cli.TerminalUtils;
+import io.trino.client.ClientSession;
+import io.trino.client.uri.TrinoUri;
+import java.io.PrintStream;
+import java.net.URI;
+import java.time.ZoneId;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import jodd.io.StringOutputStream;
+import okhttp3.logging.HttpLoggingInterceptor;
+import org.jline.terminal.Terminal;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+class TrinoQueryRunner {
+  private static final Logger LOG = LoggerFactory.getLogger(TrinoQueryRunner.class);
+
+  private QueryRunner queryRunner;
+  private Terminal terminal;
```

Comment:
Is this tool used to read information from the terminal?

Reply:
Yes